src/clr-icons/shapes/essential-shapes.ts

```diff
@@ -813,7 +827,14 @@ export const EssentialShapes: any = {
     "accessibility-2": ClrShapeAccessibility2,
     "sort-by": ClrShapeSortBy,
     "collapse-card": ClrShapeCollapseCard,
-    "expand-card": ClrShapeExpandCard
+    "expand-card": ClrShapeExpandCard,
+    "briefcase": ClrShapeBriefcase,
+    "color-picker": ClrShapeColorPicker,
+    "copy-to-clipboard": ClrShapeCopyToClipboard,
```

Comment:
shouldn't this just be `copy`?

Reply:
We already have `copy`. Maybe, it should be `copy-2` instead?

Reply:
Ok, I will keep it `copy-to-clipboard`